### PR TITLE
update note

### PR DESCRIPTION
**Pull request by susanharper-okta**
## Description:
- **What's changed?** Update note per PATEAM-451
- **Is this PR related to a Monolith release?** 

### Resolves:

* [OKTA-606209](https://oktainc.atlassian.net/browse/OKTA-606209)

### CODE REVIEW

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx score **DRAFT 06.01.22**
 
A minimum Acrolinx Score of **40** is required. The total score is an average of the subscores.

Select **Total score** to review the Acrolinx scorecard for your article. Try to increase your individual scores, ex. Correctness. Your article will be clearer and more consistent with Microsoft standards.
 
| Article | **Total score**<br />**Required:70** | Word and phrases<br />(Brand, terms)<br />Preferred: 40 | Correctness<br />(Spelling, grammar)<br />Preferred: 40 | Clarity<br />(Readability)<br />Preferred: 10 | Inclusive language<br />(+ accesibility)<br />Preferred: 10 |
|---------|:------------------------------------:|:--------------------------------------------------------:|:--------------------------------------------------------:|:---------------------------------------------:|:------------------------------------------------------------:|
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/session-cookie/main/index.md](https://github.com/okta/okta-developer-docs/blob/d2bc55b20cda33235eb4fdb0cee1e0b57cee0cab/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fsession-cookie%2Fmain%2Findex.md) | [77](https://okta.acrolinx.cloud/api/v1/checking/scorecards/f2fb2a6d-4b69-41f0-9ee0-8e43f426d6cf) | 85  | 85  | 40  | 82  |

  - The results from this response were procced by the [hugo-targets1.solution.acrolinx.sh](https://hugo-targets1.solution.acrolinx.sh) Acrolinx Cloud Host.
  - Scorecard links by default are only good for 7 days. [Link for this commit's Content Analysis Dashboard](https://okta.acrolinx.cloud/api/batch/gen.github.4b94382d-201d-42bb-bc5e-9cb4e3a9f376?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

File: packages/@okta/vuepress-site/docs/guides/session-cookie/main/index.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 title: Work with Okta session cookies
 ---
 
-<ClassicDocOieVersionNotAvailable />
+> **Note:** This document is written for Okta Classic Engine. If you are using Okta Identity Engine, see [Understand how sessions work after the upgrade](/docs/guides/oie-upgrade-sessions-api/) and the [Sessions APIs](/docs/guides/ie-limitations/main/#sessions-apis) section of the Identity Engine limitations page. See [Identify your Okta solution](https://help.okta.com/okta_help.htm?type=oie&id=ext-oie-version) to determine your Okta version.
```
Comment (timhowie-okta) [inline packages/@okta/vuepress-site/docs/guides/session-cookie/main/index.md:5]: could use "you're" to be more conversational

**Review by timhowie-okta — APPROVED** [review]
lgtm

File: packages/@okta/vuepress-site/docs/guides/session-cookie/main/index.md (hunk 1 above)
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/guides/session-cookie/main/index.md:5]:
```suggestion
> **Note:** This document is written for Okta Classic Engine. If you're using Okta Identity Engine, see [Understand how sessions work after the upgrade](/docs/guides/oie-upgrade-sessions-api/) and the [Sessions APIs](/docs/guides/ie-limitations/main/#sessions-apis) section of the Identity Engine limitations page. See [Identify your Okta solution](https://help.okta.com/okta_help.htm?type=oie&id=ext-oie-version) to determine your Okta version.
```

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx score **DRAFT 06.01.22**
 
A minimum Acrolinx Score of **40** is required. The total score is an average of the subscores.

Select **Total score** to review the Acrolinx scorecard for your article. Try to increase your individual scores, ex. Correctness. Your article will be clearer and more consistent with Microsoft standards.
 
| Article | **Total score**<br />**Required:70** | Word and phrases<br />(Brand, terms)<br />Preferred: 40 | Correctness<br />(Spelling, grammar)<br />Preferred: 40 | Clarity<br />(Readability)<br />Preferred: 10 | Inclusive language<br />(+ accesibility)<br />Preferred: 10 |
|---------|:------------------------------------:|:--------------------------------------------------------:|:--------------------------------------------------------:|:---------------------------------------------:|:------------------------------------------------------------:|
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/session-cookie/main/index.md](https://github.com/okta/okta-developer-docs/blob/254f8d9800765689f6ffe965514a9cf2b6984f90/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fsession-cookie%2Fmain%2Findex.md) | [78](https://okta.acrolinx.cloud/api/v1/checking/scorecards/dbb015d0-6060-4c4c-986f-a94e8ee40939) | 85  | 85  | 40  | 82  |

  - The results from this response were procced by the [hugo-targets1.solution.acrolinx.sh](https://hugo-targets1.solution.acrolinx.sh) Acrolinx Cloud Host.
  - Scorecard links by default are only good for 7 days. [Link for this commit's Content Analysis Dashboard](https://okta.acrolinx.cloud/api/batch/gen.github.e5e233f0-df9c-4e0f-9db6-e3bdaf02afc6?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx score **DRAFT 06.01.22**
 
A minimum Acrolinx Score of **40** is required. The total score is an average of the subscores.

Select **Total score** to review the Acrolinx scorecard for your article. Try to increase your individual scores, ex. Correctness. Your article will be clearer and more consistent with Microsoft standards.
 
| Article | **Total score**<br />**Required:70** | Word and phrases<br />(Brand, terms)<br />Preferred: 40 | Correctness<br />(Spelling, grammar)<br />Preferred: 40 | Clarity<br />(Readability)<br />Preferred: 10 | Inclusive language<br />(+ accesibility)<br />Preferred: 10 |
|---------|:------------------------------------:|:--------------------------------------------------------:|:--------------------------------------------------------:|:---------------------------------------------:|:------------------------------------------------------------:|
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/oie-upgrade-sessions-api/main/index.md](https://github.com/okta/okta-developer-docs/blob/fdc9cc2e288a6cc5af7f1e5f01a881f5c6d73575/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Foie-upgrade-sessions-api%2Fmain%2Findex.md) | [75](https://okta.acrolinx.cloud/api/v1/checking/scorecards/04cdfb26-878c-48fa-a929-e673d1138966) | 72  | 46  | 72  | 100  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/session-cookie/main/index.md](https://github.com/okta/okta-developer-docs/blob/fdc9cc2e288a6cc5af7f1e5f01a881f5c6d73575/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fsession-cookie%2Fmain%2Findex.md) | [78](https://okta.acrolinx.cloud/api/v1/checking/scorecards/6523a31a-d064-4791-a8b4-bd5c8df59122) | 85  | 85  | 40  | 82  |

  - The results from this response were procced by the [hugo-targets1.solution.acrolinx.sh](https://hugo-targets1.solution.acrolinx.sh) Acrolinx Cloud Host.
  - Scorecard links by default are only good for 7 days. [Link for this commit's Content Analysis Dashboard](https://okta.acrolinx.cloud/api/batch/gen.github.04a3ece7-f36f-49c4-982f-e5cef4c9bd9c?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

**Pull request merged**